forge-game/src/main/java/forge/game/phase/Untap.java

```diff
@@ -230,12 +236,15 @@ public boolean apply(final Card c) {
         
         // remove exerted flags from all things in play
         // even if they are not creatures
-        for (final Card c : game.getCardsInGame()) {
+        for (final Card c : game.getCardsIn(ZoneType.Battlefield)) {
```

Comment:
The little comment says "in play"... but this was checking everything in the game? Seems like this might run a bit faster this way but I wasn't sure if it was right rules-wise

Reply:
I agree, other cards should be cleaned from zone change